cli/clibase/env.go

```diff
@@ -0,0 +1,42 @@
+package clibase
```

Comment:
The naming of `environment` is abbreviated to `env` but `command` not to `cmd` bothers me. Let's make this consistent.